chat/v1/Permissions.md

```diff
@@ -36,6 +36,8 @@ Multiple or-expressions can compound, in which they effectively "multiply" the a
 A role is a collection of rules, which can be assigned to users.
 These are named and coloured for aesthetic reasons.
 
+Each guild MUST have a default role. This role MUST have the ID `0`.
```

Comment:
mention that this role shouldn't be listed in lists of roles (such as the list of guild roles, or the list of rules a user has), since it's assumed to always be there

Reply:
That should be left up to client (some people may want to see an @everyone like in Discord, some may not)

Reply:
yeah that's why it shouldn't be in the list: so the client can decide how to present it. at the top, at the bottom, as its own button, etc.

Reply:
Yes, but do we really want to add a separate endpoint to just get one role's information? The default role can be edited to change it's color / name / hoistable / pingable, so it should be included in the get guild roles / get user roles, the client can filter it itself.

Reply:
We should explain that too then